force-app/main/default/lwc/errorSettings/errorSettings.html

```diff
@@ -0,0 +1,22 @@
+<template>
+    <c-settings-save-canvas
+        component-title={labelReference.systemSettingsPageHeading}
+        onsettingseditmodechange={handleSettingsEditModeChange}
+        onsettingssaving={handleSettingsSaving}
+        onsettingssavecompleted={handleSettingsSaveCompleted}
+    >
+        <!-- Body of Contact Information Model Settings page (populates Slot in child component) -->
+        <!-- Error Settings -->
```

Comment:
is this commenting out on purpose? Just to make sure.

Reply:
yes, just to make it a little easier to find specific settings in the markup